include/sys/euiAnimator.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <nn/font/font_Util.h>
+#include <nn/ui2d/ui2d_Pane.h>

```

Comment:
I have to keep Pane because there is one specific function that uses Pane called `SetupWithPane`. Very specific use-case but it is required for eui's Animator.

Reply:
The cpp file (where that function will be implemented) can have the include, this header doesn't seem to need it from what I can tell (usually you only want to add a header include when it's needed for a function implementation or part of the class size calculations)